src/Operation/Delete.php

```diff
@@ -117,18 +117,38 @@ public function execute(Server $server)
             throw UnsupportedException::collationNotSupported();
         }
 
+        $deleteOptions = $this->createDeleteOptions();
```

Comment:
This doesn't appear to be used, since `$bulk->delete()` calls `$this->createDeleteOptions()` directly.